pkg/signup/service/signup_service_test.go

```diff
@@ -1234,13 +1241,26 @@ func (s *TestSignupServiceSuite) TestGetSignupByUsernameOK() {
 
 	us := s.newUserSignupComplete()
 	us.Name = service.EncodeUserIdentifier(us.Spec.IdentityClaims.PreferredUsername)
+	// Set the scheduled deactivation timestamp 1 day in the future
+	deactivationTimestamp := time.Now().Add(time.Hour * 24).Round(time.Second).UTC()
+	us.Status.ScheduledDeactivationTimestamp = util.Ptr(v1.NewTime(deactivationTimestamp))
 	err := s.FakeUserSignupClient.Tracker.Add(us)
 	require.NoError(s.T(), err)
 
 	mur := s.newProvisionedMUR("ted")
+	// Set the provisioned time 29 days in the past
+	provisionedTime := time.Now().Add(-time.Hour * 24 * 31).Round(time.Second)
```

Comment:
You changed the comment, but not the value
```suggestion
	provisionedTime := time.Now().Add(-time.Hour * 24 * 29).Round(time.Second)
```

Reply:
Oops, my fault. Thanks for picking this up.